src/BlueskyPostService.php

```diff
@@ -36,6 +37,14 @@ public function addFacetsFromMentionsAndLinks(Post $post): Post
         return $resultPost;
     }
 
+	public function addFacetsFromMentionsAndLinksAndTags(Post $post): Post
```

Comment:
that function name gets more and more ridiculous ;) perhaps we should break the api by renaming it to addAllFacets() at a later stage...